### ISSUE TITLE
SA1118 should not warn on second argument

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/StyleCopAnalyzers#2688]
```cs
namespace RoslynSandbox
{
    class Foo
    {
        public Foo(string text)
        {
            text.Replace(
                "A" +
                "B",
                "C" +
                "D");
        }
    }
}
```

Also `The parameter spans multiple lines` should be `The argument spans multiple lines`

**Comment by vweijsters** [comment]
I'm curious why you think that no warning should be produced. As far as I can see the behaviour matches the documented behaviour.

**Comment by JohanLarsson** [comment]
Ah, ok, should I close the issue?
Reason I wrote it was that I only saw a warning for the second argument. Also the message is technically wrong but mo biggie.
